spec/scryfall/cards_spec.rb

```diff
@@ -28,5 +28,13 @@
         end
       end
     end
+
+    context 'random' do
+      it 'returns a JSON response containing the data for a random card' do
+        VCR.use_cassette('successful random search') do
+          expect(Scryfall::Cards.random(query: ' ')['object']).to eq('card')
+        end
+      end
+    end
```

Comment:
Good job writing a test! I think maybe we should add one more that uses the query to narrow the pool of random cards so we know that aspect of it works as well.

Reply:
I did run it in irb, so I know it works I'll write up the test and push it through when I wake up.